internal/integration_test/vs/runtime.go

```diff
@@ -37,6 +37,7 @@ type Runtime interface {
 type Module interface {
 	CallI32_I32(ctx context.Context, funcName string, param uint32) (uint32, error)
 	CallI32I32_V(ctx context.Context, funcName string, x, y uint32) error
+	CallI32I32_I64(ctx context.Context, funcName string, x, y uint32) (uint64, error)
```

Comment:
revert this and the supporting changes if only used for the CGO benchmark.

Reply:
done